src/redux/selectors/index.js

```diff
@@ -0,0 +1,17 @@
+// FETCH
+const fetch_data = state => state.fetch.data;
+const fetch_word = state => state.fetch.word;
+
+// LOCAL
+const local = state => state.local;
```

Comment:
I would suggest you to have more descriptive names.

Reply:
Got IT